contrib/qubes/src/vms/fwupd_download_updates.py

```diff
@@ -90,10 +107,9 @@ def download_updates(self, url, sha):
         url - url address of the update
         """
         self.validate_vm_dirs()
-        self.arch_name = url.replace("https://fwupd.org/downloads/", "")
-        self._decrypt_update_url(url)
+        self.arch_name = os.path.basename(url)
         update_path = os.path.join(FWUPD_VM_UPDATES_DIR, self.arch_name)
-        cmd_update = ["wget", "-O", update_path, self.dec_url]
+        cmd_update = ["wget", "-O", update_path, url]
```

Comment:
Use `curl` instead?
```suggestion
        cmd_update = ["curl", "--proto", "=https", "--proto-redir", "=https", "--tlsv1.2", "--http1.1", "-sSfL", "-o", update_path, "--", url]
```

Reply:
Curl might make sense, but I don't like this strict limits on what protocol to use, as it makes it less flexible if other remotes are used, or server configuration changes. For example `--http1.1` will exclude usage of `HTTP/2`